src/role_squadheal.js

```diff
@@ -113,6 +114,6 @@ roles.squadheal.action = function(creep) {
   return true;
 };
 
-roles.squadheal.execute = function(creep) {
-  //  creep.log('Execute!!!');
-};
+//roles.squadheal.execute = function(creep) {
+//  creep.log('Execute!!!');
+//};
```

Comment:
need tooAngel validation but pretty sure it can be totally removed. I remember i removed some call in room_my if i'm right